components/builder-api/src/server/resources/profile.rs

```diff
@@ -170,6 +172,61 @@ async fn revoke_access_token(req: HttpRequest,
         Err(err) => return err.into(),
     };
 
+    let get_tokens_req = req.clone();
+
+    let tokens_response = get_access_tokens(get_tokens_req).await;
```

Comment:
If you look below on line 235, this function already gets the access tokens which are the same retrieved by `get_access_tokens`. So you should be able to remove lines 175 - 203.